usage-insights-launch.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Introducing usage monitoring
+description: 'Learn about the new Usage feature available in Xata'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/usage-insights-launch.jpg
+  alt: Xata
+author: Siobhán Meier, Ryan Slade
+date: 03-20-2022
+tags: ['launch-week', 'metrics', 'announcement']
+published: false
+slug: usage-metrics-launch
+---
+
+Usage monitoring is a new feature that visualizes usage of the Xata resources available to users. This beta release lets users explore their usage metric data at a workspace level for the first time.
```

Comment:
@alexfrancoeur do we have a "marketing friendly" product description for why this feature is awesome? Most of our internal copy is pitches this as a missing feature affecting observability and customer trust. I'm struggling to parse that into a "this is why you should be excited!" message.